### DISCUSSION TITLE
Livewire removes attribute on the model after $refresh

### DISCUSSION THREAD

**Discussion by tnt-freskim-veliu** [livewire/livewire#5096] [1. Help] [answered]
Currently, I'm having a problem with the Student model which has lessons_count attribute that is added by loadCount.
You can check the images, so you can figure out what's happening.

Here's the code that I wrote in the component:
<img width="272" alt="Screen Shot 2022-07-09 at 23 53 18" src="https://user-images.githubusercontent.com/74197546/178123825-57bb2419-0277-4598-9c1e-7011aa459380.png">

Here is the code that I have in the blade, (wire:poll.5s is just for testing, also you can call a method in the component and you will get the same result):
<img width="272" alt="Screen Shot 2022-07-09 at 23 56 58" src="https://user-images.githubusercontent.com/74197546/178123897-b082f521-1b22-4dda-bdc0-d3ea1d41fdf0.png">

Here is the UI after mount:
<img width="272" alt="Screen Shot 2022-07-09 at 23 52 49" src="https://user-images.githubusercontent.com/74197546/178123847-b8baebe3-148c-42f9-b95f-4b2297afb347.png">

And here is the UI after $refresh is called:
<img width="272" alt="Screen Shot 2022-07-09 at 23 52 40" src="https://user-images.githubusercontent.com/74197546/178123868-befc108d-ccc0-400f-9ac8-d7c4a37e6c92.png">

Thank you 😄

**Comment by luanfreitasdev** [comment]
I believe that using refresh the loadCount function should be in the render and not in the mount. Did you take this test?

**Comment by joshhanley** [comment, marked as answer]
@tnt-freskim-veliu Livewire doesn't support re-hydrating models with custom queries, such as withCount/loadCount.

See my comment here for more details https://github.com/livewire/livewire/discussions/2950#discussioncomment-1208795

Hope this helps!
